Shared/Shared/GameObject.cpp

```diff
@@ -96,15 +103,17 @@ void GameObject::serialize(NetBuffer &buffer) const {
 	buffer.write<vec3>(velocity);
 	buffer.write<vec3>(scale);
 	buffer.write<int>(id);
-	buffer.write<int>(radius);
+	// NOT SURE NEED TO WRITE BOUNDING SHAPE TO NETBUFFER
+	// SHOULD ONLY BE SERVER SIDE, PLEASE CONFIRM IN CODE REVIEW
```

Comment:
I agree. I think the collision stuff is all server-side since the client doesn't really control movement.